CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v2.3.0 2024-02-09
```

Comment:
What happened to 2.2.0?

Reply:
There's an open PR with 2.2.0 https://github.com/nxt-insurance/nxt_cop/pull/132/files

I thought it'll be merged before mine. But I'm not sure now

Reply:
I'll merge it!

Reply:
Rebased